src/controllers/Enterprise/postEnterpriseController.js

```diff
@@ -1,16 +1,15 @@
 const { Enterprise } = require("../../database");
 
 const postEnterpriseController = async (ruc, name, address) => {
-    await Enterprise.create({
-        ruc,
-        name,
-        address,
-    });
-
-    return {
-        created: true,
-        name,
-    };
+  const newEnterprise = await Enterprise.create({
+    ruc,
+    name,
+    address,
+  });
+  return {
+    created: true,
```

Comment:
I don't think it's necessary. We can say that the object was successfully created with http code 201.

Reply:
Ok. I'll resolve the conversation once the task is finished.